WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -554,7 +559,7 @@ private struct ProductsSection: View {
 
                     HStack(spacing: OrderForm.Layout.productsHeaderButtonsSpacing) {
                         scanProductButton
-                        .renderedIf(viewModel.isAddProductToOrderViaSKUScannerEnabled)
+                            .renderedIf(viewModel.isAddProductToOrderViaSKUScannerEnabled && presentationStyle == .modalOnModal)
```

Comment:
I'm planning to remove the check for `viewModel.isAddProductToOrderViaSKUScannerEnabled` next, this feature has been live for quite a while now and having the feature flag check here and through other views increases its complexity for no benefit.